contextual_lenses/pfam_family_classification/pfam_nearest_neighbors.py

```diff
@@ -64,6 +64,74 @@
 optimizer = create_optimizer(cnn_max_pool_model, learning_rate=0., weight_decay=0.)
 
 
+# Lens training: None, NN training: PF00001 - PF00100
+train_family_accessions = []
+test_family_accessions = []
+for i in range(1, 101):
+  family_name = 'PF%05d' % i
+  train_family_accessions.append(family_name)
+  test_family_accessions.append(family_name)
+
+results = pfam_nearest_neighbors_classification(encoder=optimizer.target, 
+											    train_family_accessions=train_family_accessions, 
+											    test_family_accessions=test_family_accessions)[0]
+
+print("Lens training: None, NN training: PF00001 - PF00100")
+print(results)
+print()
+
+
+# Lens training: None, NN training: PF00101 - PF00200
+train_family_accessions = []
+test_family_accessions = []
+for i in range(101, 201):
+  family_name = 'PF%05d' % i
+  train_family_accessions.append(family_name)
+  test_family_accessions.append(family_name)
+
+results = pfam_nearest_neighbors_classification(encoder=optimizer.target, 
+											    train_family_accessions=train_family_accessions, 
+											    test_family_accessions=test_family_accessions)[0]
+
+print("Lens training: None, NN training: PF00101 - PF00200")
+print(results)
+print()
+
+
+# Lens training: None, NN training: PF00001 - PF00200
+train_family_accessions = []
+test_family_accessions = []
+for i in range(1, 201):
+  family_name = 'PF%05d' % i
+  train_family_accessions.append(family_name)
+  test_family_accessions.append(family_name)
+
+results = pfam_nearest_neighbors_classification(encoder=optimizer.target, 
+											    train_family_accessions=train_family_accessions, 
+											    test_family_accessions=test_family_accessions)[0]
+
+print("Lens training: None, NN training: PF00001 - PF00200")
+print(results)
+print()
+
+
+# Lens training: None, NN training: PF00001 - PF00400
+train_family_accessions = []
+test_family_accessions = []
+for i in range(1, 401):
+  family_name = 'PF%05d' % i
+  train_family_accessions.append(family_name)
+  test_family_accessions.append(family_name)
+
+results = pfam_nearest_neighbors_classification(encoder=optimizer.target, 
+											    train_family_accessions=train_family_accessions, 
+											    test_family_accessions=test_family_accessions)[0]
+
+print("Lens training: None, NN training: PF00001 - PF00400")
```

Comment:
consider making a function which accepts a start & end pfam family, then computes the above, rather than duplicating code three times:

```
def train_and_test_pfam_families(start, end, n_train_samples):
  # Make train & test families
  ... 
  #  run pfam_nearest_neighbors_classification
  ...
  # Print results
```

Could also be used in next file